shell/platform/darwin/ios/framework/Source/FlutterTextureRegistryRelayTest.mm

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterTextureRegistryRelay.h"
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+
+#import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
+
+FLUTTER_ASSERT_ARC
+
+@interface FlutterTextureRegistryRelayTest : XCTestCase
+@end
+
+@implementation FlutterTextureRegistryRelayTest
```

Comment:
can you add a test to verify that this fixes the retain cycle?

Reply:
done.